pkg/apis/admission/register.go

```diff
@@ -27,16 +27,6 @@ const GroupName = "admission.k8s.io"
 // SchemeGroupVersion is group version used to register these objects
 var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: runtime.APIVersionInternal}
 
-// Kind takes an unqualified kind and returns a Group qualified GroupKind
```

Comment:
I would be happy to hear suggestions from the community on whether the large number of similar **unused exposed** `Kind` and `Resource` functions should be removed or retained